README.md

```diff
@@ -1,5 +1,7 @@
 # Dependency Analytics LSP Server
 
+[![Build Status](https://ci.centos.org/job/devtools-fabric8-analytics-lsp-server/100/badge/icon)](https://ci.centos.org/job/devtools-fabric8-analytics-lsp-server/100/)
```

Comment:
Link is incorrect, it is pointing a specific build(100).

Look at https://ci.centos.org/job/devtools-fabric8-analytics-lsp-server/badge/ for the right link.

[![Build Status](https://ci.centos.org/job/devtools-fabric8-analytics-lsp-server/badge/icon)](https://ci.centos.org/job/devtools-fabric8-analytics-lsp-server/)

Reply:
ohh sorry will change it

Reply:
@arajkumar PTAL